setup.cfg

```diff
@@ -29,13 +29,8 @@ install_requires =
     numpy>=1.19
     pyparsing>=2.2
     requests>=2.22
-    # force a temporary install of rad from github
     romanad
-    # fix rad version until CRDS is updated
-    #romanad==0.6.1
-    #Fix roman_datamodels during updates
-    roman_datamodels
-    #roman_datamodels==0.5.2
+    roman_datamodels==0.7.0
```

Comment:
Should this be >= ?

Reply:
Not until the cal_step and logging is fixed. All the old files are failing validation because the logging was put in as required not optional.

Reply:
This PR (and the corresponding updated input and truth files) fixes the regtest failures introduced by cal_logs:

https://github.com/spacetelescope/romancal/pull/364

Reply:
Regtests are passing on the main branch now, so it should be safe to un-pin this:

https://plwishmaster.stsci.edu:8081/job/RT/job/eslavich-dev/140/